src/condor_contrib/docker/execute/packagelist.txt

```diff
@@ -0,0 +1,44 @@
+# GNU's bc (a numeric processing language) and dc (a calculator):
+bc
```

Comment:
Do I want to know what requires bc?

Reply:
Shell scripts that don't know how to use `$(( ))` or `expr`?

I built the package list by getting a list of what gets installed by `yum groupinstall Base` and removing what's obviously unnecessary.